docs/admins/permissions-roles.rst

```diff
@@ -0,0 +1,89 @@
+:og:description: Apiary supports granular access control to different functions of the software.
+
+Permissions and Roles
+=====================
+
+Apiary supports granular access control to limit access to the software.
+Most actions require a specific permission, and permissions are grouped into roles.
+Users can be assigned any combination of roles and permissions, and their effective access is cumulative across all assigned roles and permissions.
+
+Role Descriptions
+-----------------
+
+Roles provide a quick and easy way to assign a group of permissions to a user.
+
+.. warning::
+   For service accounts using the :doc:`/admins/api/index`, assign specific permissions rather than a role.
+
+.. _admin:
+
+``admin``
+~~~~~~~~~
+This role grants access to almost all functions and should only be assigned to system administrators, developers, and other technical users. Some functions check for an associated ``admin`` role rather than a specific permission.
+
+.. _officer:
+
+``officer``
+~~~~~~~~~~~
+
+- View users, :ref:`dues transactions <Dues Transaction>`, and :doc:`payments </officers/payments/index>`
+- Create and update :ref:`dues packages <Dues Package>`
+- Create and update events
+- :doc:`Take attendance for any team or event </officers/attendance>`
+- :ref:`Record cash, check, or waiver payments <Recording an Offline Payment>`
+- :ref:`Distribute merchandise <Recording Merchandise Distribution>`
+- Manage travel
+
+.. _project-manager:
+
+``project-manager``
+~~~~~~~~~~~~~~~~~~~
+
+- View users, :ref:`dues transactions <Dues Transaction>`, and :doc:`payments </officers/payments/index>`
+- :doc:`Take attendance for any team or event </officers/attendance>`
+- :ref:`Record cash or check payments <Recording an Offline Payment>`
+- :ref:`Distribute merchandise <Recording Merchandise Distribution>`
+- Manage travel
+
+.. _team-lead:
+
+``team-lead``
+~~~~~~~~~~~~~
+
+- View users, :ref:`dues transactions <Dues Transaction>`, and :doc:`payments </officers/payments/index>`
+- :doc:`Take attendance for any team or event </officers/attendance>`
+- :ref:`Distribute merchandise <Recording Merchandise Distribution>`
+
+.. _trainer:
+
+``trainer``
+~~~~~~~~~~~
+
+- View users
+- :doc:`Take attendance for any team or event </officers/attendance>`
+
+``member`` and ``non-member``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+These are identical roles that provide baseline access to end users. They are assigned automatically based on user behavior and should not be manually attached or detached.
+
+Special Permissions
+-------------------
+
+The following permissions are not included with any roles, and must be manually assigned to a user if necessary.
```

Comment:
I thought there was another payment related one but I don't see it.

Reply:
I thought waivers were separate, but they're included with the officer role and have been forever - #965